source/_posts/2015-09-10-mqtt-different-ways-to-use-mqtt-with-home-assistant.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "MQTT: Different ways to use MQTT with Home Assistant"
+description: "This post describes three different ways to use MQTT with Home Assistant."
+date: 2015-09-10 11:19:38 +0200
+date_formatted: "September 10, 2015"
+author: Fabian Affolter
+comments: true
+categories: 
+og_image: /images/blog/2015-09-mqtt/arduino.png
+---
+
+The [MQTT](https://en.wikipedia.org/wiki/MQTT) support was added to Home Assistant recently. The [MQTT component](https://home-assistant.io/components/mqtt.html) will enable you to do all sort of things. Most likely you will use it to communicate with your devices. But Home Assistant don't care where the data is coming from or is limited to real hardware as long as there is MQTT support. This means that it doesn't matter if the data is coming from a human, a web service, or a device. 
+
+A great example is shown in a [Laundry Automation](https://home-assistant.io/blog/2015/08/26/laundry-automation-with-moteino-mqtt-and-home-assistant/) post in this blog. 
+
+This post will give you a small overview of some other possibilities on how to use MQTT with Home Assistant. 
+
+<!--more-->
```

Comment:
A nice graphic would be very nice. Adding a sceenshot of a badge wouldn't be very appealing.